src/services/BaseService.js

```diff
@@ -35,7 +35,7 @@ function makeHttpRequest(method, uri, body, headers, callback) {
     }
   };
 
-  if (body !== undefined) {
+  if (typeof body === 'object') {
```

Comment:
I solved this for you with commit 902b178de6f234ff38ee24b858350db6000f166d. Please have a look and then merge :+1: